src/app/core/services/data.service.ts

```diff
@@ -9,6 +9,11 @@ import { Observable } from 'rxjs';
 @Injectable({
   providedIn: 'root',
 })
+
+/**
+ * Responsible for pasring data related to staff and student information
```

Comment:
Typo for ‘parsing’!

Reply:
I think we can also mention that it retrieves & processes data

Reply:
Thanks for the catch!